src/views/product-integrations-landing/components/integrations-list/style.module.css

```diff
@@ -51,6 +51,7 @@
 		}
 
 		& .viewDetails {
+			color: var(--token-color-foreground-strong);
```

Comment:
Right now this just inherits 'black'. This token works for both dark and light mode.